ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -470,3 +470,8 @@ def tensor_diag_part(input, name=None):
     reshaped = ivy.reshape(input, (prod, prod))
     diagonal = ivy.diagonal(reshaped)
     return ivy.reshape(diagonal, tuple(half_shape))
+
+
+@to_ivy_arrays_and_back
+def lu_solve(lower_upper, rhs):  # ToDO
+    return ivy.solve(lower_upper, rhs)
```

Comment:
Here you go, this might help you 🙂 https://unify.ai/docs/ivy/docs/functional/ivy/linear_algebra/ivy.functional.ivy.linear_algebra.solve.html#solve